npm/vue2/patches/@vue+test-utils+1.3.0.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/node_modules/@vue/test-utils/types/index.d.ts b/node_modules/@vue/test-utils/types/index.d.ts
```

Comment:
Vue test utils was importing some types that weren't exported, and not exporting some types that we were importing. Since we bundle the types, we can get away with patching them.